src/EFCore/Metadata/ContextParameterBinding.cs

```diff
@@ -21,7 +21,7 @@ public class ContextParameterBinding : ServiceParameterBinding
         /// <param name="serviceProperties"> The associated <see cref="IServiceProperty" /> objects, or <see langword="null" />. </param>
         public ContextParameterBinding(
             Type contextType,
-            IPropertyBase[]? serviceProperties = null)
+            params IPropertyBase[]? serviceProperties)
```

Comment:
Should probably make the array non-nullable - with params we just get an empty one (here and below).

```suggestion
            params IPropertyBase[] serviceProperties)
```

Reply:
It was previously possible to pass null directly, so we should not break that.